config/locales/en/models/link_attempt.yml

```diff
@@ -0,0 +1,11 @@
+---
+en:
+  activemodel:
+    errors:
+      models:
+        link_attempt:
+          attributes:
+            maat_reference:
+              blank: Enter a maat id in the correct format
```

Comment:
I thought we can allow blank MAAT reference - remove? - if this is something else then it should be "Enter a MAAT reference"